### PR TITLE
Better management of city shield and labels

### PR DESCRIPTION
**Pull request by kent1D**

### CODE REVIEW

File: labels.mss [hunk 1]
```diff
@@ -53,18 +53,34 @@
...
+  [type='hc'],[type='ambassade'],[type='capitale'] {
+    shield-fill: @city_text;
+    shield-face-name: @bold;
+    shield-halo-fill: @halo;
+    shield-size: 12;
+    shield-text-dx: 6;
+  }
+  [type='hc'],[type='ambassade'] {
+    shield-file: url('icon/place/ambassade.svg');
+    shield-allow-overlap: true;
```
Comment (yohanboniface) [inline labels.mss]: This is quite risky, but I see the point :)
There are a bunch of glitches:
screenshot-20180216134033-291x193
screenshot-20180216134025-494x375
screenshot-20180216133057-497x327

I guess we can continue a bit by setting direction city by city, but I'm not totally convinced this is not an endless fight ;)
Other option may be to decorrelate image and label with `shield-unlock-image` and let it be more automatic.
But let's continue by manually setting the direction and see how it goes :)

File: labels.mss [hunk 2]
```diff
@@ -53,18 +53,34 @@
...
   }
-  [capital='yes'] {
+  [type='capitale'] {
     shield-file: url('icon/place/capital.svg');
   }
+  [type='consulat'],[type='consulat_general'],[type='rp'],[type='bureau'] {
```
Comment (yohanboniface) [inline labels.mss]: A few things I'd suggest:
- extrapolate categories right in SQL to keep the cartocss and XML simple/light with `CASE`, something like:

    `CASE WHEN type IN ('consulat', 'consulat_general') THEN 'intermediate' ELSE type END AS type`

- keep cartocss in english so we don't mix, and thus use the SQL to translate the categories if we want to keep the categories in French in the files

Comment (kent1D) [reply labels.mss]: try to lauch a make import_city before?

File: labels.mss (hunk 1 above)
Comment (kent1D) [reply labels.mss]: oups sorry, here should be my comment : try to launch a make import_city before?

Comment (kent1D) [reply labels.mss]: ok tou work on forte and me on piano

ldir is not used on forte at the moment, I will add it

Comment (kent1D) [reply labels.mss]: hmm no, forte use ldir also

Here is what I see

For Piano :

Piano : capture d ecran 2018-02-16 a 14 22 38


For Forte : 

Forte : capture d ecran 2018-02-16 a 14 24 02

I think `make import_city` is missing

Comment (yohanboniface) [reply labels.mss]: Yes! sorry, now it's good :)

File: labels.mss (hunk 2 above)
Comment (kent1D) [reply labels.mss]: so we have capital, embassy, intermediate

Comment (kent1D) [reply labels.mss]: and intermediate.svg and embassy.svg

File: labels.mss [hunk 3]
```diff
@@ -63,21 +63,21 @@
     shield-line-spacing: -2;
     shield-text-dx: 6;
   }
-  [type='hc'],[type='ambassade'],[type='capitale'] {
+  [type='embassy'],[type='capital'] {
```
Comment (yohanboniface) [inline labels.mss]: If we have one to one matching between types and icons, you can do something like

    shield-file: url('icon/place/[type].svg');

Comment (kent1D) [reply labels.mss]: Done

**Pull request merged**